routing/index_road_graph.cpp

```diff
@@ -90,28 +90,36 @@ void IndexRoadGraph::GetEdges(Junction const & junction, bool isOutgoing, TEdgeV
     if (!handle.IsAlive())
       MYTHROW(RoutingException, ("Can't get mwm handle for", file));
 
-    edges.emplace_back(FeatureID(MwmSet::MwmId(handle.GetInfo()), segment.GetFeatureId()),
-                       segment.IsForward(), segment.GetSegmentIdx(),
+    auto featureId = FeatureID(MwmSet::MwmId(handle.GetInfo()), segment.GetFeatureId());
+    if (!isOutgoing && m_starter.FitsStart(segment))
+    {
+      edges.emplace_back(featureId, segment.IsForward(), segment.GetSegmentIdx(),
```

Comment:
Am I right that for the edge that's being added here, |m_forward| may be set to any value? I mean, what'll happen if ```segment.IsForward()``` will be replaced by ```true``` or ```false``` here?